### PR TITLE
Fixes for section start-up issues and project directory updates

### PR DESCRIPTION
**Pull request by Yoga07**

### CODE REVIEW

File: src/node/elder_duties/key_section/transfers/mod.rs [hunk 1]
```diff
@@ -333,7 +347,40 @@ impl Transfers {
             Ok(None) => None,
             Ok(Some(event)) => {
                 self.wrapping
-                    .send(Message::NodeCmd {
+                    .send_to_client(Message::NodeCmd {
```
Comment (oetyng) [inline src/node/elder_duties/key_section/transfers/mod.rs:350]: This here is wrong, should be send to section.

File: src/node/elder_duties/key_section/transfers/replica_manager.rs [hunk 2]
```diff
@@ -120,12 +120,28 @@ impl ReplicaManager {
...
+            && self.info.section_proof_chain.len() <= 7
+            && self.info.section_proof_chain.len() != 1
+        {
+            // Safe to unwrap
+            let all_events = self.all_events().unwrap();
+            // Do nothing if we already have the same events or in case we get an empty event after initiating
+            if all_events == events || (!all_events.is_empty() && events.is_empty()) {
+                info!("Already initiated!");
+                return Ok(());
+            }
```
Comment (oetyng) [inline src/node/elder_duties/key_section/transfers/replica_manager.rs:135]: Alright, this gets us going. Nice job ironing the validations out.

Probably something fundamental there that we should look into later to make this startup easier.

**Review by oetyng — APPROVED** [review]

File: Cargo.toml [hunk 3]
```diff
@@ -37,8 +31,8 @@ pickledb = "~0.4.0"
 quick-error = "1.2.2"
 rand = "~0.7.3"
 rand_chacha = "~0.2.2"
-#sn_routing = { path="../routing" }
-sn_routing = { git = "https://github.com/bochaco/sn_routing.git", branch = "async-quic-p2p-api" }
+sn_routing = { git = "https://github.com/maidsafe/sn_routing.git" }
+self_encryption = "~0.19.3"
```
Comment (lionel1704) [inline Cargo.toml:35]: Self encryption shouldn't be a dependency here

**Review by lionel1704 — APPROVED** [review]
Just one small comment.

File: src/capacity/rate_limit.rs [hunk 4]
```diff
@@ -46,14 +46,14 @@ impl RateLimit {
...
-        let nodes = (available_nodes + full_nodes) as f64;
+        let available_nodes = (all_nodes - full_nodes) as f64;
         let supply_demand_factor = 0.001
-            + (1_f64 / available_nodes as f64).powf(8_f64)
-            + ((full_nodes + 30) as f64 / nodes).powf(88_f64);
+            + (1_f64 / available_nodes).powf(8_f64)
+            + (full_nodes as f64 / all_nodes as f64).powf(88_f64);
         let data_size_factor = (bytes as f64 / MAX_CHUNK_SIZE as f64).powf(2_f64)
             + (bytes as f64 / MAX_CHUNK_SIZE as f64);
         let steepness_reductor = prefix_len as f64 + 1_f64;
```
[the remark this answers is not in the document]
Comment (Yoga07) [reply src/capacity/rate_limit.rs:59]: Agreed, they are scaling factors for computing StoreCost, we'll be getting rid of them soon. Will take them down in the upcoming PR.

**Pull request merged**